Monika After Story/game/script-food.rpy

```diff
@@ -0,0 +1,164 @@
+    m 1hua "Be sure to pour an extra glass for me next time."
+    return
+    
+init 5 python:
+    addEvent(Event(persistent._mas_food_database,'mas_food_hotchocolate',prompt='Hot Chocolate',category=[store.mas_food.TYPE_DRINK],unlocked=True,),code='FOO')
+    
+label mas_food_hotchocolate:
+    if mas_isWinter():
+        m 1hub "That's so nice, [player]!"
+        m 4eub "There's really nothing quite like the comfort of hot chocolate when it's chilly outside, is there?"
+        m 1eub "In winter I always enjoy wrapping up in some warm clothes and unwinding with a cup myself."
+        m 1eua "It's a great way to relax."
+        m 1kua "Maybe some day when it's cold out, we could sit back and have a mug together."
+        m 1duu "That would be a dream come true, my love!"
+        
+    else:
+        m 1hua "That sounds really good right about now."
+        m 1hub "There's nothing better than a nice glass of hot cocoa after a long day."
+        m 1duu "I can't wait until we can drink some together."
+    return
+init 5 python:
+    addEvent(Event(persistent._mas_food_database,'mas_food_chocolatemilk',prompt='Chocolate Milk',category=[store.mas_food.TYPE_DRINK],unlocked=True,),code='FOO')
+    
+label mas_food_chocolatemilk:
+    m 3eub "That's wonderful, [player]."
+    m 3eub "Although I never drank regular milk too often, there was something different when you mixed chocolate with it."
+    m 1lud "..."
+    m 1rusdlb "How much do you have [player]?"
+    m 1hua "Mind pouring me a glass? ehehe~"
+    return
+    
+init 5 python:
+    addEvent(Event(persistent._mas_food_database,'mas_food_pizza',prompt='Pizza',category=[store.mas_food.TYPE_FOOD],unlocked=True,),code='FOO')
+    
+label mas_food_pizza:
+    m 1hua "Pizza is such a great treat!"
+    m 3eua "It's usually not the healthiest food, of course, but I think on occasion, it's fine to just treat yourself, you know?"
+    m 3hub "I think it's really interesting that pizza was created almost by accident."
+    m 3eub "Italian bakery workers would use excess dough and left-over ingredients from their days of work to feed the poor."
+    m 3eub "Who would have thought that they were making what would become one of the most popular foods today?"
+    m 1eub "As I'm sure you guessed I always order a vegetarian pizza myself, but I want you to know that I would never judge you if you're eating a slice with pepperoni, sausage or any other meat, [player]."
+    m 1hua "What always matters to me is that you're happy!"
+    return
+    
+init 5 python:
+    addEvent(Event(persistent._mas_food_database,'mas_food_salad',prompt='Salad',category=[store.mas_food.TYPE_FOOD],unlocked=True,),code='FOO')
+    
+label mas_food_salad:
+    m 1hua "That's great to hear, [player]!"
+    m 3eua "I'm so glad to hear that you are taking care of yourself and eating healthy."
+    m 3eua "Salads are definitely one of my personal favorite foods."
```

Comment:
You don't need to have the same expression twice in a row. Just leave the second one blank and the first one should carry over.